lib/Runtime/HostManager/HostManager.cpp

```diff
@@ -92,21 +92,36 @@ HostManager::init(std::vector<std::unique_ptr<DeviceConfig>> configs) {
 
 HostManager::~HostManager() { llvm::toString(clearHost()); }
 
-llvm::Error HostManager::addNetwork(std::unique_ptr<Module> module,
-                                    CompilationContext &cctx,
-                                    bool saturateHost) {
-  std::lock_guard<std::mutex> networkLock(networkLock_);
-  auto functions = module->getFunctions();
-  for (auto &F : functions) {
-    std::string name = F->getName();
+void HostManager::cleanupAddNetwork(llvm::ArrayRef<std::string> names) {
```

Comment:
I looked at the coverage for this PR: https://fb-glow-assets.s3.amazonaws.com/coverage/coverage-pull/3425/lib/Runtime/HostManager/HostManager.cpp.gcov.html

It looks like internals of the for loop below are not exercised. Can we make a test covering newly added code?

Reply:
@rdzhabarov  in that report blue means covered right? I think it's covered now.

Reply:
yup, blue is covered. red is not.